examples/article-access-laterpay.amp.html

```diff
@@ -233,15 +236,8 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
           </div>
         </div>
 
-        <section id="meter-notif" amp-access="views > 0" amp-access-hide class="meter-section">
-          <template amp-access-template type="amp-mustache">
-            You are viewing article {{views}} of {{maxViews}}!
-          </template>
-          <a on="tap:meter-notif.hide">[Close]</a>
-        </section>
-
         <section amp-access="NOT error AND NOT access" amp-access-hide class="login-section">
-          <a on="tap:amp-access.login">Login to read more!</a>
+          <amp-access-laterpay-list></amp-access-laterpay-list>
```

Comment:
So, I looked through this in more detail, and I have two points/questions:

1. Would the name `amp-access-laterpay-dialog` be better for this?
2. Do you have a default action that you can take if a publisher didn't specify this tag? E.g. is it possible to append the dialog to the bottom of the page? Or something like this?
3. The way custom elements are supported in AMP, this maybe better to switch to use an element with id, e.g. `<div id="amp-access-laterpay-dialog">`. WDYT? If you prefer a tag name - I'll update our validator to allow it and will ensure that AMP runtime does not try to "manage" this tag, it's all possible, but I wanted to discuss.